main.cpp

```diff
@@ -187,6 +249,7 @@ void run()
     << std::left << std::setw(12) << "Average" << std::endl;
 
   std::cout << std::fixed;
+  }
```

Comment:
One could always run the benchmark with `-np 1` (or run the non-MPI benchmark). We *could* also run the benchmark on a singe node first, and then print out the efficiency. This would probably need a bit of an overhaul in the way the driver works though. Not sure if it's worth the pain.